_sass/blocks/explore/_overview.scss

```diff
@@ -1,11 +1,25 @@
+.explore-section_top,
 .explore-section {
-	border-bottom: $base-border-size solid $dark-gray;
-	margin-bottom: $base-padding;
+	padding-bottom: $base-padding;
+	padding-top: $base-padding;	
```

Comment:
Line should be indented with spaces, not tabs<br>Line contains trailing whitespace